Gruntfile.coffee

```diff
@@ -711,12 +712,15 @@ module.exports = (grunt) ->
 
 		# Minify
 		uglify:
+			options:
+				report: "min"
```

Comment:
I think this can be dropped now since it looks looks like it's the default https://github.com/gruntjs/grunt-contrib-uglify#report